xtractmime/__init__.py

```diff
@@ -42,14 +45,10 @@ def _is_match_mime_pattern(
 def extract_mime(
     body: bytes,
     *,
-    content_types: Optional[Tuple[bytes]] = None,
+    content_types: Optional[Tuple[Union[bytes, str]]] = None,
```

Comment:
Didn’t we agree to make it only `bytes` in our last meeting?

Reply:
That's what I mentioned in my last comment above 😅. I think we agreed on input type to be (bytes, str) and output to be only bytes but I might be wrong.

Reply:
@elacuesta Do you remember?

Reply:
Not sure to be honest, but accepting only bytes will most likely simplify things.